src/GitHub.Api/ApiClientConfiguration_User.cs

```diff
@@ -1,11 +1,13 @@
+﻿using System;
+
 namespace GitHub.Api
 {
-    public partial class ApiClient : IApiClient
+    static partial class ApiClientConfiguration
```

Comment:
Yeah, I was wondering about that as well. We can think about that later I'd say.